include/seqan3/range/container/dynamic_bitset.hpp

```diff
@@ -0,0 +1,1832 @@
+     * Linear in size().
+     *
+     * ### Exceptions
+     *
+     * No-throw guarantee.
+     */
+    constexpr iterator erase(const_iterator pos) noexcept
+    {
+       return erase(pos, pos + 1);
+    }
+
+    /*!\brief Appends the given element value to the end of the container.
+     * \param value The value to append.
+     *
+     * If the new size() is greater than capacity() this is undefined behaviour.
+     *
+     * ### Complexity
+     *
+     * Constant.
+     *
+     * ### Exceptions
+     *
+     * No-throw guarantee.
+     */
+    constexpr void push_back(value_type const value) noexcept
+    {
+        assert(size() < capacity_);
+        resize(size() + 1);
+        (*this)[size() - 1] = value;
+    }
+
+    /*!\brief Removes the last element of the container.
+     *
+     * Calling pop_back() on an empty container is undefined. In debug mode an assertion will be thrown.
+     *
+     * No iterators or references except for back() and end() are invalidated.
+     *
+     * ### Complexity
+     *
+     * Constant.
+     *
+     * ### Exceptions
+     *
+     * No-throw guarantee.
+     */
+    constexpr void pop_back() noexcept
+    {
+        assert(size() > 0);
+        resize(size() - 1);
+    }
+
+    /*!\brief Resizes the container to contain count elements.
+     * \param[in] count The new size.
+     * \param[in] value Append copies of value when resizing, default = false.
+     *
+     * \details
+     *
+     * If count is greater than capacity this is undefined behaviour.
+     * If the dynamic_bitset is enlarged, bits in [0, size()) stay the same and bits in [size(), count)
+     * are set to value.
+     * If the dynamic_bitset is shrunk, bits in [0, count) stay the same and bits in [count, size()) are set to 0.
+     *
+     * ### Example
+     *
+     * \include test/snippet/range/container/dynamic_bitset_resize.cpp
+     *
+     * ### Complexity
+     *
+     * Constant.
+     *
+     * ### Exceptions
+     *
+     * No-throw guarantee.
+     */
+    constexpr void resize(size_type const count, value_type const value = false) noexcept
+    {
+        assert(count <= capacity_);
+        // Enlarging.
+        data_.bits |= value && count > size() ? ((1ULL << (count - size())) - 1) << size() : 0ULL;
+        // Set size bits.
+        data_.size = count;
+        // Shrinking.
+        data_.bits &= (1ULL << size()) - 1ULL;
+    }
+
+    /*!\brief Swap contents with another instance.
+     * \param rhs The other instance to swap with.
+     *
+     * ### Complexity
+     *
+     * Linear in the size of both containers.
+     *
+     * ### Exceptions
+     *
+     * No-throw guarantee if value_type is std::is_nothrow_copy_constructible.
+     */
+    constexpr void swap(dynamic_bitset & rhs) noexcept
+    {
+        bitfield tmp = std::move(data_);
+        data_ = std::move(rhs.data_);
+        rhs.data_ = std::move(tmp);
+    }
+
+    //!\overload
+    constexpr void swap(dynamic_bitset && rhs) noexcept
```

Comment:
don't need this

Reply:
We have this in all the containers, so why not here?

Reply:
what are you saving? The constructor and the assignment are defaulted anyway. Non one us using the swap internally. the copy-n-swap idiom is not necessary here. Hence, std::swap would work already for this class and should be the preferred way. 
Mhmm but I see know, that it is the stupid container requirement. I'll make this a discussion point in the next dev meeting

Reply:
Anyway, std::swap is not constexpr...